docs/AppFramework.md

```diff
@@ -64,34 +66,35 @@ spec:
 
 6. Apply the Custom Resource specification: `kubectl apply -f Standalone.yaml`
 
-The App Framework detects the Splunk App archive files available in the App Source locations, and deploys the apps to the standalone instance for local use. The App Framework will also scan for changes to the App Source folders based on the polling interval, and deploy updated archives to the instance. A Pod reset is triggered to install the new or modified apps.
+The App Framework detects the Splunk app or add-on archive files available in the App Source locations, and deploys them to the standalone instance path for local use. 
 
-Note: A similar approach can be used for installing apps on License Manager using it's own CR.
+The App Framework maintains a checksum for each app or add-on archive file in the App Source location. The app name and checksum is recorded in the CR, and used to compare the deployed apps to the app archive files in the App Source location. The App Framework will scan for changes to the App Source folders using the polling interval, and deploy any updated apps to the instance. For the App Framework to detect that an app or add-on had changed, the updated app must use the same archive file name as the previously deployed one. 
```

Comment:
This paragraph (&the next) appear repeated in 3 different examples: here, line 140 & line 214. Can this be consolidated, possibly outside of the examples?

Reply:
The idea from the start was to use examples, and in this doc there are three. As a reader isn't going to read the doc end-to-end, but will focus on the example that's most applicable to their needs, there is repetition of some material. If you feel the material is unimportant enough that if it was missed there would be no impact to the customer, it can be consolidated. However, as I interpret the material, these appear to be important enough to the use of the App Framework to warrant being highlighted. In that case, restating the material in each example is appropriate.

Reply:
@eddikay Thanks for clarifying the rationale behind the duplicate text, makes sense to me